CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
  * Added LH2+O2 processes to the Fuel Cells for USI and CryoTanks Support (PiezPiedPy)
  * Added Hydrogen Liquefaction and Liquid Hydrogen Evaporator processes to USI and CryoTanks Support (PiezPiedPy)
  * Fixed CryoTanks NRE in Planner and added LH2 to radial tanks for CryoTanks Support (PiezPiedPy)
+ * Reliability: mass and cost difference between standard and high quality is now relative to the part type (Gordon Dry)
```

Comment:
Most recent changes go at the top 😉 no need to change your PR though, just FYI

Reply:
thanks, I will have that in mind